tools/windows_buildenv.bat

```diff
@@ -0,0 +1,104 @@
+@ECHO OFF
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+CALL :REALPATH %~dp0\..
+SET MIXXX_ROOT=%RETVAL%
+
+IF "%PLATFORM%"=="" (
+    SET PLATFORM=x64
+)
+
+IF "%CONFIGURATION%"=="" (
+    SET CONFIGURATION=release-fastbuild
+)
+
+IF "%BUILDENV_BASEPATH%"=="" (
+    SET BUILDENV_BASEPATH="%MIXXX_ROOT%\buildenv"
+)
+
+IF "%BUILDENV_BASEPATH%"=="" (
+    SET BUILDENV_BASEPATH="%MIXXX_ROOT%\buildenv"
+)
+
+CALL :COMMAND_%1
+EXIT /B 0
+
+:COMMAND_name
+    CALL :READ_ENVNAME
+    ECHO "%RETVAL%"
+    IF "%CI%" == "true" (
+        ECHO BUILDENV_NAME=!RETVAL! >> %GITHUB_ENV%
+    )
+    GOTO :EOF
+
+:COMMAND_setup
+    CALL :READ_ENVNAME
+    SET BUILDENV_NAME=%RETVAL%
+    SET BUILDENV_PATH=%BUILDENV_BASEPATH%\%BUILDENV_NAME%
+    MD %BUILDENV_BASEPATH%
+
+    IF NOT EXIST %BUILDENV_PATH% (
+        SET BUILDENV_URL=https://downloads.mixxx.org/builds/buildserver/2.3.x-windows/%BUILDENV_NAME%.zip
+        ECHO Downloading !BUILDENV_URL!
+        BITSADMIN /transfer buildenvjob /download /priority normal !BUILDENV_URL! !BUILDENV_PATH!.zip
+        REM TODO: verify download using sha256sum?
```

Comment:
This can be done on Windows but it's inconvenient to get just the SHA256 string into a variable.

Reply:
Yes, let's just leave this as a TODO. I really don't want to do any more self-flagellation (a.k.a. writing windows batch files) right now.

Reply:
Okay. Please remove the `?` from the comment.